hapi-fees/src/main/java/com/hedera/services/pricing/BaseOperationUsage.java

```diff
@@ -0,0 +1,146 @@
+	private static final SignatureMap ONE_PAIR_SIG_MAP = SignatureMap.newBuilder()
+			.addSigPair(SignaturePair.newBuilder()
+					.setPubKeyPrefix(ByteString.copyFromUtf8("a"))
+					.setEd25519(CANONICAL_SIG))
+			.build();
+	private static final SigUsage SINGLE_SIG_USAGE = new SigUsage(
+			1, ONE_PAIR_SIG_MAP.getSerializedSize(), 1
+	);
+	private static final BaseTransactionMeta NO_MEMO_AND_NO_EXPLICIT_XFERS = new BaseTransactionMeta(0, 0);
+
+	private static final TokenOpsUsage TOKEN_OPS_USAGE = new TokenOpsUsage();
+	private static final ConsensusOpsUsage CONSENSUS_OPS_USAGE = new ConsensusOpsUsage();
+
+	/**
+	 * Returns the total resource usage for the base configuration of the given
+	 * type of the given operation.
+	 *
+	 * @param function the operation of interest
+	 * @param type the type of interest
+	 * @return the total resource usage of the base configuration
+	 */
+	UsageAccumulator baseUsageFor(HederaFunctionality function, SubType type) {
+		switch (function) {
+			case CryptoTransfer:
+				switch (type) {
+					case DEFAULT:
+						return hbarCryptoTransfer();
+					case TOKEN_FUNGIBLE_COMMON:
+						return htsCryptoTransfer();
+					case TOKEN_NON_FUNGIBLE_UNIQUE:
+						return nftCryptoTransfer();
+				}
+				break;
+			case ConsensusSubmitMessage:
+				return submitMessage();
+			case TokenFeeScheduleUpdate:
+				return feeScheduleUpdate();
+		}
+
+		throw new IllegalArgumentException("Canonical usage unknown");
+	}
+
+	private UsageAccumulator submitMessage() {
+		final var baseMeta = new BaseTransactionMeta(0, 0);
+		final var opMeta = new SubmitMessageMeta(100);
+		final var into = new UsageAccumulator();
+		CONSENSUS_OPS_USAGE.submitMessageUsage(SINGLE_SIG_USAGE, opMeta, baseMeta, into);
+		return into;
+	}
+
+	private UsageAccumulator feeScheduleUpdate() {
+		/* A canonical op */
+		final var target = TokenID.newBuilder().setShardNum(1).setRealmNum(2).setTokenNum(3).build();
+		final List<CustomFee> theNewSchedule = List.of(
+				CustomFee.newBuilder().setFixedFee(FixedFee.newBuilder()
+						.setAmount(123L)
+						.setDenominatingTokenId(target))
+						.build());
+		final var op = TokenFeeScheduleUpdateTransactionBody.newBuilder()
+				.setTokenId(target)
+				.addAllCustomFees(theNewSchedule)
+				.build();
+
+		/* The canonical usage and context */
+		final var newReprBytes = TOKEN_OPS_USAGE.bytesNeededToRepr(theNewSchedule);
+		final var grpcReprBytes = op.getSerializedSize() - FeeBuilder.BASIC_ENTITY_ID_SIZE;
+		final var opMeta = new FeeScheduleUpdateMeta(0L, newReprBytes, grpcReprBytes);
+		final var feeScheduleCtx = new ExtantFeeScheduleContext(THREE_MONTHS_IN_SECONDS, 0);
+
+		final var into = new UsageAccumulator();
+		TOKEN_OPS_USAGE.feeScheduleUpdateUsage(
+				SINGLE_SIG_USAGE,
+				NO_MEMO_AND_NO_EXPLICIT_XFERS,
+				opMeta,
+				feeScheduleCtx,
+				into);
+		return into;
+	}
+
+	private UsageAccumulator hbarCryptoTransfer() {
+		throw new AssertionError("Not implemented!");
```

Comment:
Why not provide a implementation for this case? I thought hbar transfer can be considered as the `true` base scenario for CryptoTransfer operation.

Reply:
Creating a separate issue to do this once I know Product's requirements for the base price of an NFT transfer. 😄

Reply:
Got it. Thanks for the clarification!